src/screens/CompareModels/CompareModels.js

```diff
@@ -119,72 +214,121 @@ export function CompareModels({ match, location }) {
             Python HTTP Server
           </a>
         </div>
+        <ComparisonControlsContainer>
+          <FormControl style={{ width: '12rem' }}>
+            <InputLabel focused={false}>Sort by:</InputLabel>
+            <Select value={sortType} onChange={changeSort}>
+              ><MenuItem value={SORT_TYPES.ALPHABETICAL}>State Name</MenuItem>
+              <MenuItem value={SORT_TYPES.OVERWHELMED}>
+                Hospital Overload
+              </MenuItem>
+            </Select>
+            {sortType === SORT_TYPES.OVERWHELMED && (
+              <div style={{ fontSize: 'x-small' }}>
+                ∆ between "Hospitals Overloaded" dates
+              </div>
+            )}
+          </FormControl>
+          <FormControl style={{ width: '12rem', marginLeft: '2rem' }}>
+            <InputLabel focused={false}>Filter by intervention:</InputLabel>
+            <Select
+              value={filterTypeIntervention}
+              onChange={changeInterventionFilter}
+            >
+              <MenuItem value={DEFAULT_INTERVENTION_FILTER}>
+                {DEFAULT_INTERVENTION_FILTER}
+              </MenuItem>
+              <MenuItem value={INTERVENTIONS.LIMITED_ACTION}>
+                {INTERVENTIONS.LIMITED_ACTION}
+              </MenuItem>
+              <MenuItem value={INTERVENTIONS.SOCIAL_DISTANCING}>
+                {INTERVENTIONS.SOCIAL_DISTANCING}
+              </MenuItem>
+              <MenuItem value={INTERVENTIONS.SHELTER_IN_PLACE}>
+                {INTERVENTIONS.SHELTER_IN_PLACE}
+              </MenuItem>
+            </Select>
+          </FormControl>
+        </ComparisonControlsContainer>
       </ModelSelectorContainer>
 
       <StateComparisonList
-        left={leftUrl}
-        right={rightUrl}
+        states={states}
+        leftModels={leftModels}
+        rightModels={rightModels}
         refreshing={refreshing}
+        sortFn={sortFunctionMap[sortType]}
+        filterFn={filterByIntervention}
```

Comment:
nit: I don't care much, but I think you could get rid of sortFn / filterFn props and just apply them to states directly, i.e.:
```
<StateComparisonList
  states={states.filter(filterFn).sort(sortFn)}
  ...
  />
```
Seems a little simpler to me, but this is fine too if you prefer it.